deploy/helm/sumologic/templates/logs/collector/otelcol/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.otellogs.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "sumologic.logs.collector.name.configmap" . }}
+  labels:
+    app: {{ template "sumologic.labels.app.logs.collector.configmap" . }}
+    {{- include "sumologic.labels.common" . | nindent 4 }}
+data:
+  {{- (tpl (.Files.Glob "conf/logs/collector/otelcol/config.yaml").AsConfig .) | nindent 2 }}
```

Comment:
I believe using `.Files.Get` would be enough here, given that we only provide one concrete path.

https://helm.sh/docs/chart_template_guide/accessing_files/#glob-patterns

Reply:
`.Files.Get` returns a string rather than a `Files` object, and we'd need to manually format data for the configmap instead of calling `.AsConfig`. I think this is the simpler way.

Reply:
👍 thanks for checking this